### DISCUSSION TITLE
How to Remove the Extra <span> Inside the Wrapper Rendered by next/image?

### DISCUSSION THREAD

**Discussion by Kennyt3** [vercel/next.js#73335] [Help]
### Summary

Hi there,

I'm using the next/image component in my Next.js project and noticed that it renders an extra <span> inside the main <span> wrapper. 

This extra <span> contains a placeholder <img> with aria-hidden="true", and it persists even after the main image loads.

Here's the relevant portion of the rendered output:

### Additional information

```js
<span class="image-wrapper">
  <span
    style="box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;max-width:100%"
  >
    <img
      style="display:block;max-width:100%;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0"
      alt=""
      aria-hidden="true"
      src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27384%27%20height=%27384%27/%3e"
    />
  </span>
  <img
    alt="Logo"
    src='url'
    class="logo"
  />
</span>
```

**Comment by icyJoseph** [comment]
Which Next.js version have you got installed?

**Reply by icyJoseph** [reply]
I ask because, as per version history, https://nextjs.org/docs/pages/api-reference/components/image#version-history:


Version | Changes
-- | --
v15.0.0 | decoding prop added. contentDispositionType configuration default changed to attachment.
v14.2.0 | overrideSrc prop added.
v14.1.0 | getImageProps() is stable.
v14.0.0 | onLoadingComplete prop and domains config deprecated.
v13.4.14 | placeholder prop support for data:/image...
v13.2.0 | contentDispositionType configuration added.
v13.0.6 | ref prop added.
v13.0.0 | The next/image import was renamed to next/legacy/image. The next/future/image import was renamed to next/image. A codemod is available to safely and automatically rename your imports. _**`<span>` wrapper removed**_. layout, objectFit, objectPosition, lazyBoundary, lazyRoot props removed. alt is required. onLoadingComplete receives reference to img element. Built-in loader config removed.

Since version 13.0.0, the span has been on its way out. So, unless you are using the legacy image component, or an older Next.js version, you shouldn't really have a span.

**Comment by Kennyt3** [comment]
I have the version 14.2.17 installed. Also I am using the Image component from next/image and I don’t think it is the older version

**Reply by icyJoseph** [reply]
Got a link to the repository/deployed site?

**Reply by Kennyt3** [reply]
No, I don’t. It is a private repo. Here is a screenshot of one of the components using the Image component and the package.js file. 
![IMG_2672](https://github.com/user-attachments/assets/3417d8ca-4b9f-470d-8e70-4d9751bee213)
![IMG_2671](https://github.com/user-attachments/assets/3c74b891-3a0a-4f10-a8ed-558d19e6bd79)

**Reply by icyJoseph** [reply]
That's the problem. It is literally using the `next/legacy/image` version, not the `next/image` version. 

Please read the information shared above ⏫ 

> **The next/image import was renamed to next/legacy/image**. The **next/future/image import was renamed to next/image.** A codemod is available to safely and automatically rename your imports. `<span>` wrapper removed. layout, objectFit, objectPosition, lazyBoundary, lazyRoot props removed. alt is required. onLoadingComplete receives reference to img element. Built-in loader config removed.

You'd need to rename that `next/legacy/image` import to `next/image`, but be aware that somethings might not work directly as expected. There's a reason why the developer on that codebase kept using `next/legacy/image`.
